hubbup-data.json

```diff
@@ -224,12 +224,12 @@
       {
         "org": "aspnet",
         "repo": "Universe",
-        "inclusionLevel": "AllItems"
+        "inclusionLevel": "ItemsAssignedToPersonSet"
       },
       {
         "org": "aspnet",
         "repo": "UserSecrets",
-        "inclusionLevel": "AllItems"
+        "inclusionLevel": "ItemsAssignedToPersonSet"
```

Comment:
Do you want to keep this as AllItems? Ultimately this repo isn't really used, but if there's an issue, you'd probably want to see it, no?